pywinauto/unittests/test_HwndWrapper.py

```diff
@@ -276,6 +276,14 @@ def testSendMessage(self):
         expected = 0x89 # 0x2000 + 0x40
         self.assertEqual(expected, code)
 
+    def test_send_chars(self):
+        testString = "+hello +world"
```

Comment:
We also need to call `dlg.minimize()` to test it in more realistic conditions, because `send_chars` is supposed to be silent (i.e. working for even minimized window).